notebooks/backwards_compatability.ipynb

```diff
@@ -0,0 +1,985 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Backwards Compatability Examples with Different Protocols\n",
```

Comment:
Extending notebook to cover gRPC  will be useful

Reply:
done